fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -512,23 +512,22 @@ private static SearchParameter getSearchParameterInternal(Map<String, Map<String
                     log.fine(String.format(UNSUPPOTED_EXPR_NULL, parameter.getType(), parameter.getCode().getValue()));
                 }
             }
-
         }
-
+        
```

Comment:
extra space?

Reply:
courtesy of eclipse i guess.  we have these all over our codebase.